### ISSUE TITLE
Overfitting on S3DIS during training

### ISSUE THREAD

**Issue by QasimKhan5x** [dvlab-research/Stratified-Transformer#63]
Hi, I am training stratified transformer with the same settings as yours except that I used `max_batch_points: 600000` and `batch_size: 24` with 3 GPUs. I got max mIOU of 0.70111 on validation. But the validation loss is as follows:
![Capture](https://user-images.githubusercontent.com/29240497/195380735-ad67a9b7-d3c3-4409-9d34-cb1fab24629c.PNG)

The training loss is decreasing smoothly on the other hand. Why is the model overfitting so bad?

**Comment by QasimKhan5x** [comment]
Ok I'll try the default config and get back to you.

**Comment by X-Lai** [comment]
But if you use different batch size, you also should modify learning rate and other related parameters accordingly.

**Comment by QasimKhan5x** [comment]
I kept the batch size at 8 and used 4 GPUs. The validation mIoU seems to peak around 70 again but the same validation loss curve is observed. Did you observe this during training?

**Comment by X-Lai** [comment]
Not yet. I got the best validation model in the last 20 epochs, and the val miou continues to increase as shown in our training log (released in the README.md file).

**Comment by QasimKhan5x** [comment]
I am getting the following curves (this is my second run with 8 batch size)

![image](https://user-images.githubusercontent.com/29240497/195984700-5a6fca92-0dd0-43f3-8830-2d42592c5079.png)

**Comment by X-Lai** [comment]
Thank you for pointing out this observation.
Actually, I am not aware of this before. But here are some hints. 1. Although the loss_val could increase later, it seems the miou_val continues to increase. The difference between these two metrics is a little weird. 2. Before 60 epochs, learning rate is fixed, so this may cause the overfitting issue. But whether the trainning is normal, we still need to see the curve of the last 40 epochs.

**Comment by X-Lai** [comment]
But overall, I think once the validation miou is strong enough, the training is normal, even though there is something unstable within.

**Comment by xindeng98** [comment]
@X-Lai Thank you for your work！ I noticed that you did not mention the number of parameters in your paper. Can you please tell me the number of parameters for the model on s3dis?

**Comment by bututoubaobei-0108** [comment]
Can you please give the environment configuration including the OS, Python version, Cuda version, GCC, and other python packages versions? Thank you very much for your help

**Comment by QasimKhan5x** [comment]
Sorry I am not working on this project anymore, so I don't have those details
